ci/ansible/roles/pulp-populate/tasks/main.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install packages for population task
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=latest"
+  with_items:
+    - python34
+    - python34-setuptools
+    - gcc
+
+- name: Ensure services are running
```

Comment:
Nitpick: Can the same term ("started") be used in both the task description and body?